packages/office-ui-fabric-react/src/components/Button/Button.scss

```diff
@@ -351,7 +351,7 @@ $button-commandButtonHeight: 40px;
   background-color: transparent;
   color: $ms-color-neutralSecondary;
   padding: 0;
-  min-width: auto;
+  min-width: 0;
```

Comment:
oh ok! I see. May want to add a comment since it looks unnecessary at first glance.